ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/recovery/RecoveryModule.java

```diff
@@ -34,4 +34,18 @@ public interface RecoveryModule
      */
 
     public void periodicWorkSecondPass ();
+
+    /**
```

Comment:
Why not just

```
     * Called by the RecoveryManager after the completion of the second pass.
     *
     * @return true if there is work left to do, false otherwise
```

to match with the style used to document the other two methods in this interface. The extra verbosity gives the impression that there are some extra subtleties involved.

Reply:
It works for me. Thanks